spesmilo/electrum
# add regtest support

See #4241 for meta discussion

1. Add a `--regtest` option
2. Change the human readable part for bech32 addresses from `tb` to `bcrt`
3. Set an empty checkpoints parameter: `CHECKPOINTS = []`
4. Set the genesis block hash to `0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206`
5. Create a new `servers_regtest.json` only `localhost` as the server and use it for setting `DEFAULT_SERVERS`
6. Make sure that `--regtest` and `--testnet` are exclusive
